docs/api_v2.md

```diff
@@ -0,0 +1,105 @@
+# cAdvisor Remote REST API
+
+cAdvisor exposes its raw and processed stats via a versioned remote REST API:
+
+`http://<hostname>:<port>/api/<version>/<request>`
+
+This document covers the detail of version 2.0. All resources covered in this version are read-only.
+
+NOTE: v2.0 is still a work in progress.
+
+## Version information
+
+Software version for cAdvisor can be obtained from version endpoint as follows:
+`/api/v2.0/version`
+
+## Machine Information
+
+The resource name for machine information is as follows:
+
+`/api/v2.0/machine`
+
+The machine information is returned as a JSON object containing:
+
+- Number of schedulable logical CPU cores
```

Comment:
I feel the [docker remote API](https://docs.docker.com/reference/api/docker_remote_api_v1.17/) documentation is pretty useful. That is the basis for my comment.